.github/workflows/test-build-deploy.yml

```diff
@@ -9,10 +9,10 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     container:
-      image: quay.io/cortexproject/build-image:update-golang-1.14.9-eb0c8d4d2
+      image: quay.io/cortexproject/build-image:upgrade-build-image-debian-491e60715-WIP
     steps:
       - name: Checkout Repo
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
```

Comment:
I assume this build image fixes old Git version problem .There is a mention in .github/README.md about this that should be updated.